ast/dml.go

```diff
@@ -2605,3 +2605,24 @@ func (n *SplitOption) Restore(ctx *RestoreCtx) error {
 	}
 	return nil
 }
+
+type FulltextSearchModifier int
+
+const (
+	FulltextSearchModifierNaturalLanguageMode = 0
+	FulltextSearchModifierBooleanMode         = 1
+	FulltextSearchModifierModeMask            = 0xF
+	FulltextSearchModifierWithQueryExpansion  = 1 << 4
```

Comment:
(why `1 << 4`? 😅)

Reply:
In case we want to support more modes compare to MySQL later on.